src/renderer/src/pages.js

```diff
@@ -176,26 +178,32 @@ const pages = {
     inspect: new InspectPage({
         label: "Inspect",
         icon: inspectIcon,
+        group: "Workflow",
     }),
     preview: new PreviewPage({
         label: "Neurosift",
         icon: neurosiftIcon,
+        group: "Workflow",
     }),
     uploads: new UploadsPage({
         label: "Uploads",
         icon: uploadIcon,
+        group: "Workflow",
     }),
     tutorial: new TutorialPage({
         label: "Tutorial",
         icon: tutorialIcon,
+        group: "Documentation",
     }),
     docs: new DocumentationPage({
         label: "Documentation",
         icon: documentationIcon,
+        group: "Documentation",
     }),
     contact: new ContactPage({
         label: "Contact Us",
         icon: contactIcon,
+        group: "Documentation",
```

Comment:
```suggestion
        group: "Help",
    }),
    docs: new DocumentationPage({
        label: "Documentation",
        icon: documentationIcon,
        group: "Help",
    }),
    contact: new ContactPage({
        label: "Contact Us",
        icon: contactIcon,
        group: "Help",
```

Minor naming suggestion; on the new tab, it feels a little redundant to have a listing "Documentation/Documentation"

Or an alternative improvement would be to call the second documentation something else, but I have fewer ideas about that

Reply:
I agree about changing "Documentation" to something else. How about "Resources" or "Reference"?

Reply:
Just updated with latest push! Though I like "Resources" more than "Help". 

@CodyCBakerPhD What do you think?

Reply:
Sounds good